components/builder-api/src/server/resources/pkgs.rs

```diff
@@ -1634,82 +1368,3 @@ fn download_response_for_archive(archive: &PackageArchive,
         .append_header((http::header::CACHE_CONTROL, cache_hdr))
         .streaming(rx_body.map(|s| Ok::<_, Infallible>(s)))
 }
-
-async fn has_circular_deps(req: &HttpRequest,
```

Comment:
I've removed this as our discussions but I'm still concerned that we may need it. What will be in place to prevent the creation of circular dependencies in the future? Something in hab autobuild?